lib/beaker-pe/install/pe_utils.rb

```diff
@@ -448,6 +455,29 @@ def deploy_frictionless_to_master(host)
           end
         end
 
+        # Check for availability of required network resources
+        # @param [Array<Host>] hosts
+        # @param [Array<String>] network_resources, defaults to NETWORK_RESOURCES
+        #
+        # @example
+        #   verify_network_resources(hosts, network_resources)
+        #
+        # @return nil
+        #
+        # @api private
+        #
+        def verify_network_resources(hosts, network_resources = NETWORK_RESOURCES)
+          logger.notify("Checking the availability of network resources.")
+          hosts.each do |host|
```

Comment:
Should we also check that each host is available to each host? This would check for conditions like network segmentation or vmpooler reaping a vm we're using.

Reply:
Oh, that's a great idea!